### PR TITLE
83 lights

### LINKED ISSUE #72: Moving of track
- [ ] Build trackpiece class
- [ ] Link to GUI
- [ ] Link to step (in order to accelerate)
- [ ] Take care of textures

### LINKED ISSUE #83: Revision of camera perspective and light settings
The textures on the walls are very dark, maybe a relocation of camera/lights will help.

< user-interface

### PR DESCRIPTION
**Pull request by thervh70**
will close #83 

This branch contains a simple light model, it serves as a proof of concept
It should be improved in the branch that will close  issue 72.

< feature

### CODE REVIEW

File: src/main/java/nl/tudelft/ti2206/group9/gui/GameScene.java [hunk 1]
```diff
@@ -1,6 +1,9 @@
 package nl.tudelft.ti2206.group9.gui;
 
-import static nl.tudelft.ti2206.group9.ShaftEscape.OBSERVABLE;
+/**
+ * @author Robin, Maarten
+ */
+import javafx.event.EventHandler;
```
Comment (mpsijm) [inline src/main/java/nl/tudelft/ti2206/group9/gui/GameScene.java]: Unused import. Also, the Javadoc containing `@author` was already present at lines 30-33.

<layout,documentation

**Comment by mpsijm** [comment]
Small pull request, but it doesn't matter, the functionality is looking good :D Of course the LightRenderer needs refactoring, but like you said, this will be done in #72 . 
Just fix the one comment I had on your diff and this is ready to be merged :)

<review,feature,documentation

**Comment by thervh70** [comment]
I've fixed the small mistake. I guess it was caused by a merge or something.
Please continue reviewing.

< review, layout

**Comment by MathiasMeuleman** [comment]
I also reviewed.
If you indeed refactor in a later iteration, this is fine.

Will merge

<review

**Pull request merged**